source/packages/services/commands/src/commands/commands.controller.ts

```diff
@@ -132,17 +132,29 @@ export class CommandsController implements interfaces.Controller {
             `commands.controller uploadCommandFile: in: commandId:${commandId}, fileId:${fileId}`
         );
 
-        const busboy = new Busboy.default({ headers: req.headers });
+        const busboy = new (Busboy.default as any)({ headers: req.headers });
         const saveTo = path.join(this.tmpDir, `${commandId}_${fileId}`);
         let hasError: string;
 
         await new Promise((resolve, reject) => {
-            busboy.on('file', async (_fieldName, stream, _filename, _encoding, _mimeType) => {
-                stream.pipe(fs.createWriteStream(saveTo));
-                stream.on('limit', () => {
-                    hasError = 'filesSizeLimit';
-                });
-            });
+            busboy.on(
```

Comment:
Can you add a comment explaining what's in these bigger changes so it's clear how this maintains existing functionality?

Reply:
See https://github.com/aws/aws-connected-device-framework/pull/210/files/3ea5837443cd6221cdfa10f4c8b47bddc7719564#r1601914368 for explanation